operator/v1/0000_70_cluster-network-operator_01.crd.yaml

```diff
@@ -284,6 +284,9 @@ spec:
                         v6InternalSubnet:
                           description: v6InternalSubnet is a v6 subnet used internally by ovn-kubernetes in case the default one is being already used by something else. It must not overlap with any other subnet being used by OpenShift or by the node network. The size of the subnet must be larger than the number of nodes. The value cannot be changed after installation. Default is fd98::/48
                           type: string
+                        GatewaySubnet:
```

Comment:
did you manually edit this or did this get auto-generated? `gatewaySubnet` is what I'd expect...

Reply:
I did that manually